source/Test/StepBro.Core.Test/Parser/Expression/TestDynamicObject.cs

```diff
@@ -97,11 +97,11 @@ public void CallDynamicMethodWithIdentifier()
         [TestMethod, Ignore]    // Identifier as a type has not been implemented yet.
         public void CallDynamicMethodReturningIdentifier()
         {
-            Assert.AreEqual(true, ParseAndRun<Identifier>(
-                "obj.Christianson()",
-                nameof(DummyDynamicObject) + " obj = " + nameof(DummyDynamicObject) + ".NewInitialized();",
-                varDummyClass: true));
-            Assert.AreEqual(0, ExecutionHelper.RuntimeErrors.Errors.Count);
+            //Assert.AreEqual(true, ParseAndRun<Identifier>(
```

Comment:
out-commented

Reply:
The test is ignored, but for some reason it started to give compiler error. Maybe test-adapter has changed.

Reply:
Consider writing that in a comment then :)